packages/cubejs-testing/birdbox-fixtures/driver-test-data/extends/postgresql/CommonCustomers.js

```diff
@@ -0,0 +1,11 @@
+import { sql } from './Customers.sql';
+
+cube(`CommonCustomers`, {
+  sql: sql('_type_'),
+  dimensions: {
+    customerId: {
+      sql: 'customer_id',
+      type: 'string',
+    },
+  },
+});
```

Comment:
📓 I just copied something from 'extends' to target directory and updating content of target file